internal/common/ingest/ingestion_pipeline.go

```diff
@@ -2,11 +2,13 @@ package ingest
 
 import (
 	"context"
+	"errors"
+	"fmt"
 	"sync"
 	"time"
 
 	"github.com/apache/pulsar-client-go/pulsar"
-	"github.com/pkg/errors"
+	pkgerrors "github.com/pkg/errors"
```

Comment:
do we actually need both "errors" and "github.com/pkg/errors"?  I'm guessing we could just have the latter.